### DISCUSSION TITLE
Idea for keystrokes

### DISCUSSION THREAD

**Discussion by scribblepleb** [keiwando/toonsquid#42] [Ideas]
Hi! I’m a (very) new user of ToonSquid, and I had a bit of an idea. When I found out ToonSquid lets you use keystrokes, I absolutely lost my mind! BUT there is something I wish I could do.
You can press a key to select a tool. Something I like about a few programs I use that also have this feature is that if you hold down the key for long enough, it switches back to the previous tool you were using when the key is released. So, if I were using the brush tool, and then pressed a key to switch to the fill tool, I could just hold it down while I colored my drawing. Then when I released the key, it would switch back to the brush tool.
I thought this could be a useful feature! That’s all from me! I hope I can contribute to this community more in the future :]

**Comment by keiwando** [comment]
That definitely sounds useful. Thanks for the suggestion!
